frontend/packages/client/src/pages/Proposal.js

```diff
@@ -325,7 +334,7 @@ export default function ProposalPage() {
           </div>
         </div>
       )}
-      {confirmingVote && castingVote && String(castVote) && (
+      {confirmingVote && castingVote && castVote && (
```

Comment:
maybe doing `castVote !== null` since it's initialised with null ? an option could be a 0

Reply:
Yes - that is the correct fix here.